src/recipe/test/e2e/__snapshots__/updateRecipe.controller.e2e-spec.ts.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Recipe Controller - Update should throw validation error when property description is 1 1`] = `
+Object {
+  "error": "Bad Request",
+  "message": Array [
+    "description must be a string",
+  ],
+  "statusCode": 400,
+}
+`;
+
+exports[`Recipe Controller - Update should throw validation error when property imageURL is 1 1`] = `
+Object {
+  "error": "Bad Request",
+  "message": Array [
+    "imageURL must be a string",
+  ],
+  "statusCode": 400,
+}
+`;
+
+exports[`Recipe Controller - Update should throw validation error when property ingredients is 1 1`] = `
+Object {
+  "error": "Bad Request",
+  "message": Array [
+    "nested property ingredients must be either object or array",
+  ],
+  "statusCode": 400,
+}
+`;
+
+exports[`Recipe Controller - Update should throw validation error when property name is 1 1`] = `
```

Comment:
I like to add a description to test that is uses like that. Then you could place it int test description so the resulting snapshot would say: 
`Recipe Controller - Update should throw validation error when property name is not string`
(in your case in validationTestsStructure)